metaflow/client/core.py

```diff
@@ -1123,6 +1124,202 @@ def _iter_filter(self, x):
         # exclude private data artifacts
         return x.id[0] != "_"
 
+    def _get_task_for_queried_step(self, flow_id, run_id, query_step):
```

Comment:
Could this function simply be replaced by `Step(flow_id/run_id/step_name).task`? It doesn't seem to bring much more to it.

Reply:
Wasn't aware that this returns the first available task.